### PR TITLE
Adapt lvgl for using ft6206 model touch screen[stm32f469-st-disco]

### PR DESCRIPTION
**Pull request by liYony**
## 拉取/合并请求描述：(PR description)

[
完成 stm32f469-st-disco BSP的另一种触摸芯片ft6206对接lvgl
]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality
- [x] 本拉取/合并使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](../documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](../documentation/contribution_guide/coding_style_en.txt)

### CODE REVIEW

File: bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c [hunk 1]
```diff
@@ -5,20 +5,25 @@
...
 #include <rtdevice.h>
+#include <touch.h>
 #include <lcd_port.h>
 
 #define DBG_TAG    "LVGL.port.indev"
 #define DBG_LVL    DBG_INFO
 #include <rtdbg.h>
 
-/* Include the package header files you are using */
+/* Import the rt_hw_xxx_init function of the package you are using */
```
Comment (mysterywolf) [inline bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c:20]:
```suggestion
```

File: bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c [hunk 2]
```diff
@@ -5,20 +5,25 @@
...
 #ifdef BSP_USING_TOUCH_FT6X36
-#include "ft6236.h"
+extern int rt_hw_ft6236_init(const char *name, struct rt_touch_config *cfg,  rt_base_t pin);
 #endif /* BSP_USING_TOUCH_FT6X36 */
+#ifdef BSP_USING_TOUCH_FT6206
+extern int rt_hw_ft6206_init(const char *name, struct rt_touch_config *cfg);
+#endif /* BSP_USING_TOUCH_FT6206 */
```
Comment (mysterywolf) [inline bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c:26]: 需要改成头文件
```suggestion
extern int rt_hw_ft6236_init(const char *name, struct rt_touch_config *cfg,  rt_base_t pin);
#else defined(BSP_USING_TOUCH_FT6206)
extern int rt_hw_ft6206_init(const char *name, struct rt_touch_config *cfg);
#endif /* BSP_USING_TOUCH_FT6X36 */
```

File: bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c [hunk 3]
```diff
@@ -43,6 +48,10 @@ static void input_read(lv_indev_drv_t *indev_drv, lv_indev_data_t *data)
     data->point.x = read_data->y_coordinate;
     data->point.y = LCD_HEIGHT - read_data->x_coordinate;
 #endif /* BSP_USING_TOUCH_FT6X36 */
+#ifdef BSP_USING_TOUCH_FT6206
```
Comment (mysterywolf) [inline bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c:51]: 用 #else defined
形式

File: bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c [hunk 4]
```diff
@@ -68,10 +77,13 @@ static int lv_hw_touch_init(void)
 {
     struct rt_touch_config cfg;
 
-#ifdef BSP_USING_TOUCH_FT6X36
     cfg.dev_name = BSP_TOUCH_I2C_BUS_NAME;
+#ifdef BSP_USING_TOUCH_FT6X36
     rt_hw_ft6236_init(TOUCH_DEV_NAME, &cfg, BSP_TOUCH_I2C_RESET_PIN);
 #endif /* BSP_USING_TOUCH_FT6X36 */
+#ifdef BSP_USING_TOUCH_FT6206
```
Comment (mysterywolf) [inline bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c:84]: 同上

**Review by mysterywolf — CHANGES_REQUESTED** [review]

**Review by mysterywolf — APPROVED** [review]

File: bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c [hunk 5]
```diff
@@ -5,19 +5,23 @@
...
+ * 2022-07-08     liYony       Add FT6206
  */
 #include <lvgl.h>
 #include <rtdevice.h>
+#include <touch.h>
 #include <lcd_port.h>
 
 #define DBG_TAG    "LVGL.port.indev"
-#define DBG_LVL    DBG_INFO
+#define DBG_LVL    DBG_LOG
```
Comment (mysterywolf) [inline bsp/stm32/stm32f469-st-disco/applications/lvgl/lv_port_indev.c:17]: 一般INFO就可以 LOG不是默认状态

**Review by Guozhanxin — APPROVED** [review]

**Pull request merged**